respx/models.py

```diff
@@ -108,6 +115,27 @@ def build_response(
         return httpx_response
 
 
+class Call(NamedTuple):
+    request: httpx.Request
+    response: Optional[httpx.Response]
+
+
+class CallList(list):
+    def __iter__(self) -> Generator[Call, None, None]:
+        yield from super().__iter__()
+
+    @classmethod
+    def from_unittest_call_list(cls, call_list: "_CallList") -> "CallList":
+        return cls(Call(request, response) for (request, response), _ in call_list)
+
+    @property
+    def last(self) -> Optional[Call]:
+        if self:
+            return self[-1]
+
+        return None
```

Comment:
I think this is neater as a one-liner...

```suggestion
        return self[-1] if len(self) else None
```